akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -36,6 +36,7 @@ akka.cluster.sharding {
   buffer-size = 100000
 
   # Timeout of the shard rebalancing process.
+  # Additionally, if an entity doesn't handle the stopMessage after `handoff-timeout` it will be stopped forcefully
   handoff-timeout = 60 s
```

Comment:
if we wait the same amount as the full handoff-timeout the rebalance will be aborted anyway. Makes sense to derive it from this property but perhaps make the receiveTimeout `(handoff-timeout - 5.seconds).max(1.second)`

Reply:
Done